client/src/assets/icons/index.tsx

```diff
@@ -11,7 +11,7 @@ import Shield from './Shield';
 import TensorflowIcon from './TensorflowIcon';
 import Algorithm from './Algorithm';
 
-const generateIconComponent = (superBlock, className) => {
+const generateIconComponent = (superBlock: any, className: any) => {
```

Comment:
What you can do here is pull `iconMap` out of the function, then use it to create a new type `SuperBlock` via

```ts
type SuperBlock = keyof typeof iconMap
```
then that type can be used for `superBlock` and put TypeScript at ease.

Reply:
Interesting. Thanks!